README.rst

```diff
@@ -103,9 +108,7 @@ structure you get from executing ``sphinx-quickstart``.
 
 
 To get Sphinx-Gallery into your project we have to extend the Sphinx
-``doc/conf.py`` file with:
-
-.. code-block:: python
+``doc/conf.py`` file with::
```

Comment:
I'm just curious. Why do you make bash code blocks now explicit, make the python blocks now un-explicit?